recorder/src/main/resources/tracking.js

```diff
@@ -73,6 +73,7 @@ jsflight.TrackHash = function(event) {
         console.log(e);
     } finally {
         jsflight.eventId++;
+        jsflight.rebindAllHandlers();
```

Comment:
Well, after further investigating, I found, that the most appropriate way to detect DOM changes is to use [MutationOserver](https://dom.spec.whatwg.org/#interface-mutationobserver). But its' [browser support](http://caniuse.com/#feat=mutationobserver) make me sad. To detect DOM changes in IE9-10 we can use [Mutation Events](https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Mutation_events). But it's deprecated, not fully supported across browsers. The only thing I can suggest is to use `setInterval(..)`